booth.spec

```diff
@@ -166,6 +166,8 @@ echo "%%run_build_tests set to %run_build_tests; skipping tests"
 %dir /usr/lib/ocf/lib
 %dir /usr/lib/ocf/lib/booth
 %dir %{_sysconfdir}/booth
+%dir %{_var}/lib/booth
+%dir %{_var}/lib/booth/cores
```

Comment:
Would it make sense to add %attr defining the exact expected permissions like:
%dir %attr (750, %{uname}, %{gname}) %{_var}/lib/booth
%dir %attr (750, %{uname}, %{gname}) %{_var}/lib/booth/cores

?